companion-select-text-insert/info.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Inserts selected text from web companion addon and appends current URL below current line ",
+  "identifier": "companion-select-text-insert",
+  "script": "selected-text-below.qml",
+  "authors": ["@v3ss0n"],
+  "platforms": ["linux", "macos", "windows"],
+  "version": "1.0.1",
+  "minAppVersion": " 20.11.7",
+  "description" : "This script inserts selected text from webpage and appends the URL below current line.\n Requires  Web companion browser extensions : https://github.com/qownnotes/web-companion/"
```

Comment:
Still too many whitespaces.
s/\n Requires  Web companion browser extensions :/\nRequires web companion browser extensions:/

Reply:
Please use: `This script inserts the selected text from a webpage and appends the URL below the current line.\nRequires the web companion browser extension <a href='https://github.com/qownnotes/web-companion/'>web-companion</a>`

Reply:
Need a formatter for QML lol
fixing

Reply:
That's json, not qml. 😉